dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -391,7 +439,16 @@ def diff(self, other_profile: NumericStatsMixin, options: Dict = None) -> Dict:
                 other_profile.variance,
                 other_profile.match_count,
             ),
+            "psi": self._calculate_psi(
+                self.mean,
+                self.sum,
+                self._stored_histogram["histogram"],
+                other_profile.mean,
+                other_profile.sum,
+                other_profile._stored_histogram["histogram"],
+            ),
```

Comment:
simply adding `psi` as a new `key` in the `differences `dict` that is returned from the `def diff()` method